src/fclone.js

```diff
@@ -20,7 +20,7 @@ function fclone(obj, refs) {
   }
 
   // typed array Int32Array etc.
-  if (obj.subarray  && typeof obj.subarray === 'function' && /^[A-Z][A-Za-z\d]+Array$/.test(Object.prototype.toString.call(obj))) {
+  if (typeof obj.subarray === 'function' && /[A-Z][A-Za-z\d]+Array/.test(Object.prototype.toString.call(obj))) {
```

Comment:
```
> Object.prototype.toString.call(new Uint8Array())
'[object Uint8Array]'
```

It wasn't correct, as there's a literal `]` after the Array word...  Figure that if it has the subarray function and it has the word Array in the object string output, it's close enough...